### ISSUE TITLE
Genre and Location are not resolving

### ISSUE THREAD

**Issue by jcoyne** [curationexperts/alexandria-v2#537]
See: http://ucsb.curationexperts.com/lib/ark:/99999/fk4sf30x0v
accession number:    mss292-g00200

The RDF looks correct.  Check that the indexer is resolving the labels:
```
   <http://www.rdaregistry.info/Elements/w/#formOfWork.en> <http://vocab.getty.edu/aat/300128371>,
     <http://vocab.getty.edu/aat/300046300>,
     <http://vocab.getty.edu/aat/300138239> .
```

**Comment by acozine** [comment]
This may be caused by #536 - I have not been able to ingest the McPeak collection objects yet. Once we fix 536 and reingest this may go away.

**Comment by jcoyne** [comment]
The show page had the wrong field. Fixed by 81eee307db6aba80921885003ec8c06e172b5f83
